### ISSUE TITLE
Sales Data

### ISSUE THREAD

**Issue by bnedavies** [Gizra/IBLI#238]
Where will we be able to access the sales data on the webplatform?

**Comment by niryariv** [comment]
I don't think we defined this yet. We can add a page for this. I assume this would admin-accessible only?

**Comment by bnedavies** [comment]
yes, admin accessible only.

**Comment by bnedavies** [comment]
Let us know when something is ready to see here...or what kind of inputs you need from us to design

**Comment by niryariv** [comment]
We haven't worked on it yet, since we don't know what data needs to be shown. I assume it's a table with user info - which fields should be there?

**Comment by bnedavies** [comment]
Will send a sample of the fields to pull and then maybe if it's possible to do graphs/tables for a few items?

**Comment by niryariv** [comment]
fields + charts: https://drive.google.com/a/gizra.com/folderview?id=0B7Lzf69iarhdYVRRc3dGM0Jra28&usp=sharing

Sales Transaction Platform API: https://docs.google.com/document/d/1T-MEuSM7e1gMrKglcUoVX8OtbstjI_NUwfUjCFkqZAw/edit?usp=sharing

@bnedavies from initial look it seems this will require modifications to the Transaction Platform API, to be done by Amos' team.

**Comment by bnedavies** [comment]
@niryariv @amosgichamba Really?  The excel sheet is a report that is taken straight from the sales platform, so it should have been in the API...

**Comment by niryariv** [comment]
It's in the system, but not exposed in the API (per the doc I have). We didn't know this data will be needed since we didn't have the detailed spec for this page, but it shouldn't be hard to add.

**Comment by bnedavies** [comment]
@amosgichamba what data was exposed in the API?

**Comment by bnedavies** [comment]
@amosgichamba please add what is needed for us to be able to pull this sales data into the webplatform...it should already be there....
